utils/cookiejarparser.go

```diff
@@ -0,0 +1,114 @@
+package utils
+
+// Code from https://github.com/ssgelm/cookiejarparser under MIT License
```

Comment:
Any reason we are copying this code rather than using it as a dependency?

Reply:
Seemed very small and not sure if it would be maintained. So felt it is better to copy instead of import

Reply:
If we are not modifying any part of the dependency code, let's introduce the dependency.
Copying code in here is a new pattern and I'd recommend not deviating from current practices (other and future maintainers will wonder why this exception).

Reply:
Done